Session4/ArrOperationTest.java

```diff
@@ -0,0 +1,90 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+
+public class ArrOperationTest {
+
+	private int[] inputArray;
+	private int x;
+	private int y;
+	private int expectedResult[];
+	ArrOperation obj = new ArrOperation();
+	
+	@Test
+	public void maxMirrorTest1() {
+		int []inputArray= {1, 2, 3, 8, 9, 3, 2, 1};
+		assertEquals(3,obj.maxMirror(inputArray));  
+	}
+	@Test
+	public void maxMirrorTest2() {
+		int []inputArray= {1, 2, 3, 8, 9, 3, 2, 1};
+		assertEquals(3,obj.maxMirror(inputArray));  
+	}
+	@Test
+	public void maxMirrorTest3() {
+		int []inputArray= {1, 2, 3, 8, 9, 3, 2, 1};
+		assertEquals(3,obj.maxMirror(inputArray));
+	}
+		
+	
+	@Test
+	public void noofClumpsTest1() {
```

Comment:
Rename the method. it should be something like `testNoOfClumps_WithExpectedOutput`

Reply:
done sir